internal/gensnippets/gensnippets.go

```diff
@@ -99,6 +103,50 @@ func Generate(rootDir, outDir string, apiShortnames map[string]string) error {
 	return nil
 }
 
+// GenerateSnippetsDirs takes in specified modules in rootDir and outputs their examples in outDir.
```

Comment:
Mention how dirs should be used. Maybe if none are passed it just does for all?

Reply:
Added an additional comment before the function.